web/wp-content/themes/dctx-v2/assets/bower_components/jquery/src/var/fnToString.js

```diff
@@ -0,0 +1,7 @@
+define( [
+	"./hasOwn"
+], function( hasOwn ) {
+	"use strict";
+
+	return hasOwn.toString;
+} );
```

Comment:
:no_entry_sign: **Error**: Closing parenthesis of a multi-line function call must be on a line by itself (*PEAR.Functions.FunctionCallSignature.CloseBracketLine*).